jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>Is there a particular reason this method requires counts per 10,000 normalization? We introduce in the normalization chapter CPM, scran and sctransform. It's fine to use any of them, we just need to add an explanation for the choice here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>No, we can use any type of normalization. We are actually not using the log transformed data from MuSiC, it's just for the initial visualization. We can maybe remove this part completely if you think it's not important.</p>

Reply:
<p>Indeed we cannot use other normalizations :)</p><p>Because of the regression, we should only use linear transformation only. </p>

Reply:
<p>This is CPM btw</p>

Reply:
<p>Should we really change this? This is just for the visualization. I think it's okay.</p>